packages/open-payments/src/client/incoming-payment.ts

```diff
@@ -89,9 +90,34 @@ export const createIncomingPaymentRoutes = (
   }
 }
 
+export interface UnauthenticatedIncomingPaymentRoutes {
+  get(args: UnauthenticatedResourceRequestArgs): Promise<IncomingPayment>
```

Comment:
This should actually be a new type of IncomingPayment we return, something like `PublicIncomingPayment` that only returns what's defined in the spec